scripts/mk_util.py

```diff
@@ -649,6 +650,7 @@ def display_help(exit_code):
     print("  --java                        generate Java bindings.")
     print("  --ml                          generate OCaml bindings.")
     print("  --python                      generate Python bindings.")
+    print("  --use-cffi                    use cffi instead of ctypes for Python bindings.")
```

Comment:
@forward-wfw The coresponding option for the CMake build system is missing. I don't see how your changes are supposed to work in the CMake build system.